models/calendar.yml

```diff
@@ -304,13 +304,14 @@
 
     - 7:45 PM:
         type: Talk
-        emoji: 🔍
-        title: How to Navigate Large Codebases When You're Used To Simple Projects
-        description: TBD
+        emoji: 🔍💻💥
+        title: Honey, I blew up the code!
+        description: For newer developers who've only worked on personal projects with small codebases, making the jump to working on a company's codebase or an open source project can be daunting. The codebase looks huge!
```

Comment:
I believe this you'll need to pipe-escape this! Like so:

```yml
description: |
  For newer developers who've only...
```

Reply:
Done :-)